src/lib/viewers/__tests__/BaseViewer-test.js

```diff
@@ -568,20 +568,42 @@ describe('lib/viewers/BaseViewer', () => {
             expect(base.resize).to.be.called;
         });
 
-        it('should show annotations', () => {
+        it('should show annotations and toggle annotations mode to NONE if discoverability FF is not enabled', () => {
```

Comment:
Seems like we could have a parameterized test for this as well based on `enableAnnotationsDiscoverability` and the called annotation `mode`